### DISCUSSION TITLE
Remove categories from palette

### DISCUSSION THREAD

**Discussion by trixtan** [eclipse/windowbuilder#781] [Q&A] [answered]
Is it possible to remove all default categories and have only my custom categories displayed?

Thank you
Kind regards
Enrico

**Comment by ptziegler** [comment, marked as answer]
You can do so via the `.*.wbp-palette-commands.xml` files.

For example, in order you to remove Swing categories, you need to create a `org.eclipse.wb.swing.wbp-palette-commands.xml` file in your `wbp-meta` folder. For testing purposes, you can add it to your project but in general, it needs to be part of the classpath.

The following snippet shows how to remove the "System" category.
```
<?xml version="1.0" encoding="UTF-8"?>
<commands>
	<removeCategory id="org.eclipse.wb.swing.system"/>
</commands>
```

When opening the editor, it then looks as follows:

![image](https://github.com/eclipse-windowbuilder/windowbuilder/assets/70652978/4e747f42-345b-4b25-bb28-febaf5f164cf)

**Reply by trixtan** [reply]
Thank you!!!
